test/integration/framework/etcd.go

```diff
@@ -221,5 +218,5 @@ func EtcdMain(tests func() int) {
 
 // GetEtcdURL returns the URL of the etcd instance started by EtcdMain.
 func GetEtcdURL() string {
-	return etcdURL
```

Comment:
`GetEtcdURL` returns the "current working url",  if the env variable doesn't have an etcd the code runs a custom etcd and this function returns that url

Reply:
nevermind, we set the env variable